recipes/kmc-smudge-pairs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "kmc_smudge_pairs" %}
+{% set version = "3.1.2rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gkanogiannis/KMC/archive/v{{ version }}.tar.gz
+  sha256: d6acc23643e3764a5ce6e3ec4cd1790592f36d992e44dc927baf0cbf14842f46
+
+build:
+  skip: true  # [not linux]
+  script: bash build_static.sh
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - make
+  host:
+    - zlib
+    - bzip2
+  run:
+    - zlib
+    - bzip2
```

Comment:
This should be handled by run_exports.
```suggestion
```

Reply:
I moved these libraries to the run_exports section of build.